ui/lib/kv/addon/components/page/secret/metadata-edit.hbs

```diff
@@ -1,6 +1,63 @@
-<KvPageHeader @breadcrumbs={{@breadcrumbs}} @model={{@model}} @pageTitle="Edit Secret Metadata" />
+<KvPageHeader @breadcrumbs={{@breadcrumbs}} @model={{@metadata}} @pageTitle="Edit Secret Metadata" />
```

Comment:
I _think_ we just need to pass the model to this component if there's any filtering happening in this view?

Reply:
You're right. I keep forgetting this. I even made a note to myself—that I didn't read—and everything 🙈